src/apps/backend/modules/share-tasks/rest-api/share-task-serializer.ts

```diff
@@ -0,0 +1,11 @@
+//backend\modules\share-tasks\rest-api\share-task-serializer.ts
+import { SharedTask } from '../types';
+
+export const serializeSharedTaskAsJSON = (sharedTask: SharedTask): unknown => ({
+  id: sharedTask.id,
+  task: sharedTask.task,
+  account: sharedTask.account,
+  title: sharedTask.title,
+  description: sharedTask.description,
+  username: sharedTask.username,
+});
```

Comment:
title and description should be part of the task and the username should be part of the account